bin/capsule.js

```diff
@@ -869,11 +888,59 @@ const addFilesToS3Bucket = async (projectName, bucketName) => {
   });
 }
 
+/**
+ *
+ * @method getCloudFrontDistID
+ *
+ * @param {String} bucketName
+ *
+ * @return {String} distId
+ *
+ */
+const getCloudFrontDistID = async (bucketName) => {
```

Comment:
not something that needs fixing, but I wanted to mention that `params` doesn't look like is needed

This function could also be written as such
```js
const getCloudFrontDistID = (bucketName) => new Promise((resolve, reject) => {
  cfr.listDistributions({}, (err, data) => {
    if (err) {
      logIfVerbose(`${err} , ${err.stack}`);
      reject(err);
    } else {
      resolve(extractDistId(data, bucketName));
    }
  }
}
```

Reply:
Ahh yeah, when we were doing dev, we were testing the params (hence the var being there). As per your comment this can be cleaned up.

Thanks!